cluster/addons/calico-policy-controller/calico-etcd-service.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    k8s-app: calico-etcd
+    kubernetes.io/cluster-service: "true"
+  name: calico-etcd
+  namespace: kube-system
+spec:
+  clusterIP: 10.0.0.17
```

Comment:
Sorry for not catching this. We parametrize the same field in kube-dns service because it needs to fall into the service subnet. This may not work if people are not using the default service subnet. I think we can defer the fix for a follow up though.